### DISCUSSION TITLE
Cannot add registry key with script, elevate to admin issue?

### DISCUSSION THREAD

**Discussion by moose999** [amidaware/tacticalrmm#1451] [General]
I need a PowerShell script to add a registry key. 

I am using the New-Item PowerShell command.

I keep geting the error "PermissionDenied"

I have tried checking the box "Run As User (Windows only)" against the script, and the currently logged in user is a local admin on the machine, but this has not changed anything.

When I run the script locally in an elevated PowerShell window, all works as expected, so I think the script is fine.

How can I get a PowerShell script to run as  / elevate to admin so that it can add registry keys?

Many thanks!

**Comment by wh1te909** [comment]
please paste the powershell script you are using and follow the issue template so we know what version of TRMM you're on and what version/OS the agent is so we can attempt to reproduce.

**Comment by dinger1986** [comment]
if you are using run as user then it wont be elevated as UAC will block it, if you dont tick the box it should work fine
